jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/test/query/TestQueryHints.java

```diff
@@ -60,7 +60,7 @@
 
     private final static int propertyTimeout = 3099;
 
-    @Emf(name = "defaultEMF", classes = { Employee.class }, createTables = DDLGen.DROP_CREATE)
+    @Emf(name = "defaultEMF", classes = { Employee.class })
```

Comment:
I believe this was intentional to allow running single test... What about adding `createTables = DDLGen.DROP_CREATE` with calls to `populate` to tests, which you are trying to fix?

Reply:
Yes, I noticed this issue in JSE tests as well and I was slowly introducing the following structure:

``` java
EntityManager em = emf.createEntityManager();
try {
    ...
} finally {
    if (em.getTransaction().isActive()) {
        em.getTransaction().rollback();
    }
    if(em.isOpen()) {
        em.close();
     }
}
```

My thought was, if `em` == null, then an exception will have been thrown from `createEntityManager()` causing that to fail. That exception should be thrown before the `try` and fail the test. If it makes it into the finally block, then `em` != null since it passed the `createEntityManager()` invocation. Thoughts?